core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/RowDenormalizerAggregator.java

```diff
@@ -87,6 +104,11 @@ public void groupBy(StructuredRecord record, Emitter<String> emitter) throws Exc
         throw new IllegalArgumentException(
           String.format("Keyfield '%s' does not exist in input schema %s", keyField, record.getSchema()));
       }
+      StructuredRecord.Builder droppedRecordBuilder = StructuredRecord.builder(errorRecordSchema);
```

Comment:
So we are using recordCounter as the key? But that is unique within only one reducer right? Will the number of reducers always be one for the RowDenormalizer plugin? If that is not the case, these values can be overridden, isn't it?